.github/workflows/test_on_push.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
     strategy:
       matrix:
-        tarantool-version: ["1.10", "2.3", "2.4", "2.5"]
+        tarantool-version: ["1.10", "2.1", "2.2", "2.6"]
```

Comment:
Please, add a comment why exactly these versions was chosen. It's very important.

Reply:
Comment should be follwing:

```
1.10 - separate tuple-merger module (https://github.com/tarantool/tuple-merger)
2.1 - there is no merger
2.2 - there is a merger but it doesn't work (https://github.com/tarantool/tarantool/issues/4954)
2.3+ - OK
```